app/org/sagebionetworks/bridge/models/schedules/TaskScheduler.java

```diff
@@ -26,11 +27,12 @@
     
     protected DateTime getScheduledTimeBasedOnEvent(Schedule schedule, Map<String, DateTime> events) {
         // If no event is specified, it's enrollment by default.
-        String eventId = schedule.getEventId();
-        if (eventId == null) {
-            eventId = "enrollment";
+        String eventIdsString = schedule.getEventId();
```

Comment:
So if schedule is delayed 7 days, it's delay 7 days after enrollment, delay 7 days after survey finished event... not quite what we want.